docs/sources/microsoft-365/README.md

```diff
@@ -89,12 +92,25 @@ NOTE: that `Mail.ReadBasic` affords only access to email metadata, not content/a
 NOTE: These are all 'Application' scopes, allowing the proxy itself data access as an application,
 rather than on behalf of a specific authenticated end-user ('Delegated' scopes).
 
+### Single Entra ID Application for Multiple Connections
+
+Our [AWS example](https://github.com/Worklytics/psoxy-example-aws/tree/main) supports using a SINGLE Entra ID application for multiple connections,
+instead of one for each. This could be ease management, but requires that you determine the superset of scopes needed across all connectors you wish to use and create
```

Comment:
```suggestion
instead of one for each. This could ease management, but requires that you determine the superset of scopes needed across all connectors you wish to use and create
```